### PR TITLE
added actions on link buttons

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx [hunk 1]
```diff
@@ -126,14 +134,37 @@ const ValidatorsList = ({ onCheckboxClickHandler, selectedValidators, selectUnse
...
+    GoogleTagManager.getInstance().sendEvent({
+      category: 'external_link',
+      action: 'click',
+      label: 'Open Beaconcha',
+    });
+    window.open(`${ENV().BEACONCHA_URL}/validator/${publicKey}`);
+  };
+
+  const openExplorer = (publicKey: string) => {
+    GoogleTagManager.getInstance().sendEvent({
```
Comment (alexstolr) [inline src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx:152]: Why do we need to add a GoogleTagManager event here? Does anyone look at the google dasboard?

File: src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx [hunk 2]
```diff
@@ -126,14 +134,37 @@ const ValidatorsList = ({ onCheckboxClickHandler, selectedValidators, selectUnse
   //   });
   // };
 
+  const copyToClipboard = (publicKey: string) => {
+    navigator.clipboard.writeText(publicKey);
+    notificationsStore.showMessage('Copied to clipboard.', 'success');
+  };
+
+  const openBeaconcha = (publicKey: string) => {
```
Comment (alexstolr) [inline src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx:142]: openBeaconcha and openExplorer should be the same function .something like openLink(url, googleTagData)

**Review by alexstolr — CHANGES_REQUESTED** [review]

File: src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx:152]: @IlyaVi we need it?

Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx]: removed for now, will work on this later

File: src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx]: done

**Review by IlyaVi — APPROVED** [review]

**Review by alexstolr — APPROVED** [review]

**Pull request merged**